src/nanosvg.h

```diff
@@ -252,7 +252,7 @@ static void nsvg__parseElement(char* s,
 {
 	const char* attr[NSVG_XML_MAX_ATTRIBS];
 	int nattr = 0;
-	char* name;
+	char* tagName;
```

Comment:
this fixes a compilation warning?  might wanna try a different compiler!

Reply:
`name` was later shadowed here https://github.com/kasper93/nanosvg/blob/3ab25342afd43a478863598a96f0e5a310022d9d/src/nanosvg.h#L282 hance the name change to avoid variable shadowing.